README.md

```diff
@@ -19,6 +19,7 @@ Then follow the prompts!
 ### Prerequisites
 
 - [node and npm](https://nodejs.org/en) (npm ≥ 5.2.0)
+- [Python 3.6+](https://www.python.org/)
```

Comment:
I believe this is only needed for Mac users - could we note that?
```suggestion
- [Python 3.6+](https://www.python.org/) For MacOS
```

Reply:
Aptos CLI requires Python 3.6+, which is mentioned in the [Aptos dev docs](https://aptos.dev/en/build/cli/install-cli/install-cli-windows#ensure-you-have-python-36-installed-by-running-python3---version). When I set it up on a brand new Windows OS (with only Node.js and npm installed), I ran into errors due to the missing Python 3.6+. Some Linux-based systems also default to Python 2, causing similar issues.

I think it would be helpful to add a general note in the requirements section about needing Python 3 pre-installed. This could save new devs some troubleshooting time!

Please also review this PR to update the Prerequisites section in the Aptos dev docs: https://github.com/aptos-labs/developer-docs/pull/533.